exchange/exchange.go

```diff
@@ -235,3 +243,74 @@ func (e *Exchange) R() *Replication {
 func (e *Exchange) Index() *Index {
 	return e.idx
 }
+
+// ImportAddress from a hex encoded private key to use as default on the exchange instead of
+// the auto generated one. This is mostly for development and will be reworked into a nicer command
+// eventually
+func (e *Exchange) ImportAddress(ctx context.Context, pk string) {
```

Comment:
These methods should be on the wallet interface in the wallet directory

Reply:
ImportKey & ExportKey are still present in the wallet ([here](https://github.com/myelnet/pop/blob/ag/improve-default-fil-key-ux/wallet/wallet.go#L354)), these are just wrappers that were previously in the node package that I moved in the exchange.
Since they are only wrapper, if we only want them in the Wallet package, I will have to change their args (in the interface too):
`ImportKey(_ context.Context, k *KeyInfo)` to `ImportKey(_ context.Context, path string)`
`ExportKey(_ context.Context, addr address.Address)` to `ExportKey(_ context.Context, addr, path string)`

Reply:
Ah sorry then you should move this logic to the node methods

Reply:
Actually I was able to put everything in the wallet package, I just changed the signature of ImportKey/ExportKey from the Driver interface

Reply:
@gallexis not really a fan of this approach, I think we need to decouple logic so the wallet module could be its own separate package and still be reusable by other projects for example. In that case we don't want to force disk writes when exporting the key of decoding from hex when importing a key. Using a generic key type allow more interoperability down the line.

Reply:
The idea is for the exchange to be usable with a different wallet implementation too for example

Reply:
hum yes you're right, I'll make some change 👍

Reply:
should be fixed